Obsidian/Net/Packets/Play/CloseContainerPacket.cs

```diff
@@ -13,9 +14,44 @@ public partial class CloseContainerPacket : IClientboundPacket, IServerboundPack
 
     public async ValueTask HandleAsync(Server server, Player player)
     {
-        if (WindowId == 0)
+        if (WindowId == 0 || (player.OpenedContainer is not IBlockEntity tileEntity))
             return;
 
-        await server.Events.InvokeContainerClosedAsync(new ContainerClosedEventArgs(player) { Container = player.OpenedContainer! });
+        var position = tileEntity.BlockPosition;
+
+        var block = await player.World.GetBlockAsync(position);
+
+        if (block == null)
+            return;
+
+        if (block.Is(Material.Chest))
+        {
+            await player.client.QueuePacketAsync(new BlockActionPacket
+            {
+                Position = position,
+                ActionId = 1,
+                ActionParam = 0,
+                BlockType = block.BaseId
+            });
+
+            await player.SendSoundAsync(SoundEffectBuilder.Create(SoundId.BlockChestClose)
+                .WithSoundPosition(position.SoundPosition)
+                .Build());
+        }
+        else if (block.Is(Material.EnderChest))
+        {
+            await player.client.QueuePacketAsync(new BlockActionPacket
+            {
+                Position = position,
+                ActionId = 1,
+                ActionParam = 0,
+                BlockType = block.BaseId
+            });
+            await player.SendSoundAsync(SoundEffectBuilder.Create(SoundId.BlockEnderChestClose)
+                .WithSoundPosition(position.SoundPosition)
+                .Build());
+        }
+
+        player.OpenedContainer = null;
```

Comment:
Why was this reverted 🤨

Reply:
Git probably messed up when merging with master